### PR TITLE
add bmp180

### PR DESCRIPTION
**Pull request by Prry**
add atmospheric sensor package for bmp180

### CODE REVIEW

File: peripherals/sensors/bmp180/Kconfig [hunk 1]
```diff
@@ -0,0 +1,36 @@
+
```
Comment (armink) [inline peripherals/sensors/bmp180/Kconfig]: sensor 里的 Kconfig 要改一下

Comment (Prry) [reply peripherals/sensors/bmp180/Kconfig]: 已经添加。谢谢

File: peripherals/sensors/bmp180/package.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  [
+    {
+      "version": "v1.0.0",
+      "URL": "https://github.com/Prry/rtt-bmp180.git",
+      "filename": "bmp180-v1.0.0.zip",
+      "VER_SHA": "b904bd31fc3c376bb6f9522db67f6d3fb70abc98"
+    },
+    {
+      "version": "latest",
+      "URL": "https://github.com/Prry/rtt-bmp180.git",
```
Comment (armink) [inline peripherals/sensors/bmp180/package.json]: @SummerLife 检查一下这个软件包为啥没有同步过去

**Comment by Prry** [comment]
在ENV执行在线更新包，确实没有把包更新下来。包的描述文件“Kconfig”和“package.json”，我又多次进行本地测试，是OK的。



------------------&nbsp;原始邮件&nbsp;------------------
发件人: "notifications"<notifications@github.com&gt;; 
发送时间: 2020年3月18日(星期三) 晚上10:40
收件人: "RT-Thread/packages"<packages@noreply.github.com&gt;; 
抄送: "潘锐"<445395670@qq.com&gt;; "Author"<author@noreply.github.com&gt;; 
主题: Re: [RT-Thread/packages] add bmp180 (#581)





 
@armink commented on this pull request.
 
 
In peripherals/sensors/bmp180/package.json:
 &gt; +  "license": "Apache-2.0", +  "repository": "https://github.com/Prry/rtt-bmp180", +  "icon": "unknown", +  "homepage": "https://github.com/Prry/rtt-bmp180", +  "doc": "unknown", +  "site":  +  [ +    { +      "version": "v1.0.0", +      "URL": "https://github.com/Prry/rtt-bmp180.git", +      "filename": "bmp180-v1.0.0.zip", +      "VER_SHA": "b904bd31fc3c376bb6f9522db67f6d3fb70abc98" +    }, +    { +      "version": "latest", +      "URL": "https://github.com/Prry/rtt-bmp180.git",  
@SummerLife 检查一下这个软件包为啥没有同步过去

File: peripherals/sensors/bmp180/package.json (hunk 2 above)
Comment (SummerLife) [reply peripherals/sensors/bmp180/package.json]: 我看了下镜像的情况，这个软件包的镜像是大写的

**Comment by SummerLife** [comment]
> 在ENV执行在线更新包，确实没有把包更新下来。包的描述文件“Kconfig”和“package.json”，我又多次进行本地测试，是OK的。
> […](#)
> ------------------&nbsp;原始邮件&nbsp;------------------ 发件人: "notifications"<notifications@github.com&gt;; 发送时间: 2020年3月18日(星期三) 晚上10:40 收件人: "RT-Thread/packages"<packages@noreply.github.com&gt;; 抄送: "潘锐"<445395670@qq.com&gt;; "Author"<author@noreply.github.com&gt;; 主题: Re: [RT-Thread/packages] add bmp180 (#581) @armink commented on this pull request. In peripherals/sensors/bmp180/package.json: &gt; + "license": "Apache-2.0", + "repository": "https://github.com/Prry/rtt-bmp180", + "icon": "unknown", + "homepage": "https://github.com/Prry/rtt-bmp180", + "doc": "unknown", + "site": + [ + { + "version": "v1.0.0", + "URL": "https://github.com/Prry/rtt-bmp180.git", + "filename": "bmp180-v1.0.0.zip", + "VER_SHA": "b904bd31fc3c376bb6f9522db67f6d3fb70abc98" + }, + { + "version": "latest", + "URL": "https://github.com/Prry/rtt-bmp180.git", @SummerLife 检查一下这个软件包为啥没有同步过去 — You are receiving this because you authored the thread. Reply to this email directly, view it on GitHub, or unsubscribe.

把镜像关了应该就可以了，镜像现在是每天早上九点同步。

File: peripherals/sensors/bmp180/package.json (hunk 2 above)
Comment (SummerLife) [reply peripherals/sensors/bmp180/package.json]: 看错了，不是大小写的情况

**Comment by SummerLife** [comment]
> > 在ENV执行在线更新包，确实没有把包更新下来。包的描述文件“Kconfig”和“package.json”，我又多次进行本地测试，是OK的。
> > […](#)
> > ------------------ 原始邮件 ------------------ 发件人: "notifications"<[notifications@github.com](mailto:notifications@github.com)>; 发送时间: 2020年3月18日(星期三) 晚上10:40 收件人: "RT-Thread/packages"<[packages@noreply.github.com](mailto:packages@noreply.github.com)>; 抄送: "潘锐"<[445395670@qq.com](mailto:445395670@qq.com)>; "Author"<[author@noreply.github.com](mailto:author@noreply.github.com)>; 主题: Re: [RT-Thread/packages] add bmp180 (#581) @armink commented on this pull request. In peripherals/sensors/bmp180/package.json: > + "license": "Apache-2.0", + "repository": "https://github.com/Prry/rtt-bmp180", + "icon": "unknown", + "homepage": "https://github.com/Prry/rtt-bmp180", + "doc": "unknown", + "site": + [ + { + "version": "v1.0.0", + "URL": "https://github.com/Prry/rtt-bmp180.git", + "filename": "bmp180-v1.0.0.zip", + "VER_SHA": "b904bd31fc3c376bb6f9522db67f6d3fb70abc98" + }, + { + "version": "latest", + "URL": "https://github.com/Prry/rtt-bmp180.git", @SummerLife 检查一下这个软件包为啥没有同步过去 — You are receiving this because you authored the thread. Reply to this email directly, view it on GitHub, or unsubscribe.
> 
> 把镜像关了应该就可以了，镜像现在是每天早上九点同步。

现在同步到镜像不是实时的，第二天再看就好了，当天测试的时候需要把镜像功能关掉。

**Pull request merged**